### PR TITLE
Scan config trend

### PR DESCRIPTION
**Pull request by swaterkamp**
**Checklist**:

- [X] Tests
- [X] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,13 +25,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
 
 ### Fixed
+- Fix showing ScanConfig trends [#1582](https://github.com/greenbone/gsa/pull/1582) [#1554](https://github.com/greenbone/gsa/pull/1554)
 - Show HostsCount at SectionTitle [#1576](https://github.com/greenbone/gsa/pull/1576)
 - Fixed creating target from hosts [#1575](https://github.com/greenbone/gsa/pull/1575)
 - Fixed CVSSBaseCalculatorv2 [#1572](https://github.com/greenbone/gsa/pull/1572)
 - Fix title in Operating Systems table of Reports [#1567](https://github.com/greenbone/gsa/pull/1567)
 - Fix displaying only usable report formats at report download [#1565](https://github.com/greenbone/gsa/pull/1565)
 - Fix handling schedule_periods ("once" option) in TaskDialog [#1563](https://github.com/greenbone/gsa/pull/1563)
-- Fix showing inactive VerifyIcon at ReportFormats detailspage [#1554](https://github.com/greenbone/gsa/pull/1554)
```
Comment (bjoernricks) [inline CHANGELOG.md]: Why did you remove the link? Was this by intention?

File: gsa/src/gmp/models/__tests__/scanconfig.js [hunk 2]
```diff
@@ -119,7 +119,7 @@ describe('ScanConfig model tests', () => {
     const scanConfig = new ScanConfig(elem);
 
     expect(scanConfig.families.count).toEqual(42);
-    expect(scanConfig.families.trend).toEqual('1');
+    expect(scanConfig.families.trend).toEqual(1);
```
Comment (bjoernricks) [inline gsa/src/gmp/models/__tests__/scanconfig.js]: the new constant should be used for the test here too
```suggestion
    expect(scanConfig.families.trend).toEqual(SCANCONFIG_TREND_DYNAMIC);
```

File: gsa/src/gmp/models/scanconfig.js [hunk 3]
```diff
@@ -52,6 +55,8 @@ export const openVasScanConfigsFilter = config =>
 export const ospScanConfigsFilter = config =>
   config.scan_config_type === OSP_SCAN_CONFIG_TYPE;
 
+const parseTrend = trend => parseInt(trend);
```
Comment (bjoernricks) [inline gsa/src/gmp/models/scanconfig.js]: Can be simplified

```suggestion
const parseTrend = parseInt;
```

File: CHANGELOG.md (hunk 1 above)
Comment (swaterkamp) [reply CHANGELOG.md]: No idea, how this happened. Maybe a leftover from the merge conflict I had to deal with...

File: gsa/src/gmp/models/__tests__/scanconfig.js [hunk 4]
```diff
@@ -134,7 +135,7 @@ describe('ScanConfig model tests', () => {
     };
     const res = {
       count: 42,
-      trend: '1',
+      trend: SCANCONFIG_TREND_DYNAMIC,
```
Comment (bjoernricks) [inline gsa/src/gmp/models/__tests__/scanconfig.js]:
```suggestion
      trend: '1',
```

Comment (bjoernricks) [reply gsa/src/gmp/models/__tests__/scanconfig.js]: That one was just fine :)

Tests are always `some plain input => expected output`. So if we get a string '1' from the backend we expect that our model converts this to `SCANCONFIG_TREND_DYNAMIC`

**Comment by bjoernricks** [comment]
A test is failing :-/
